static/styles/subscriptions.css

```diff
@@ -832,14 +832,6 @@ h4.stream_setting_subsection_title {
         }
     }
 
-    .inner-box {
-        margin: 20px;
-    }
-
-    .stream_section {
-        padding-left: 2px;
-    }
-
```

Comment:
Since, both of these classes `inner-box` and `stream_section` exists only inside the `subscription_settings` in    `#subscription_overlay` ; I've added them in nested CSS to avoid duplicity.